expression-language.md

```diff
@@ -40,27 +40,27 @@ This document is a working draft.
 
 CloudEvents SQL expressions (also known as CESQL) allow matching of CloudEvent attributes against complex expressions
 that lean on the syntax of Structured Query Language (SQL) `WHERE` clauses. Using SQL-derived expressions for message
-filtering is in widespread implementation use because the Java Message Service (JMS) message selector syntax also leans
-on SQL. Note that neither the SQL standard (ISO 9075) nor the JMS standard nor any other SQL dialect are used as a
-normative foundation or constrain the expression syntax defined in this specification, but the syntax is informed by
+filtering has widespread implementation usage because the Java Message Service (JMS) message selector syntax also leans
+on SQL. Note that neither the SQL standard (ISO 9075) nor the JMS standard nor any other SQL dialect is used as a
+normative foundation or to constrain the expression syntax defined in this specification, but the syntax is informed by
 them.
 
 CESQL is a _[Total pure functional programming language][total-programming-language-wiki]_ in order to guarantee the
 termination of the evaluation of the expression. It features a type system correlated to the [CloudEvents type
-system][ce-type-system] and it features boolean and arithmetics operations, as well as built-in functions for string
+system][ce-type-system], and it features boolean and arithmetic operations, as well as built-in functions for string
 manipulation.
 
 The language is not constrained to a particular execution environment, which means it might run in a source, in a
-producer, in an intermediary, and it can be implemented using any technology stack.
+producer, or in an intermediary, and it can be implemented using any technology stack.
 
-The CloudEvents Expression Language assumes the input always includes, but it's not limited to, a single valid and type
-checked CloudEvent instance. An expression MUST NOT mutate the value of the input CloudEvent instance, nor any of the
-other input values. The evaluation of an expression observes the concept of [referential
+The CloudEvents Expression Language assumes the input always includes, but is not limited to, a single valid and 
+type-checked CloudEvent instance. An expression MUST NOT mutate the value of the input CloudEvent instance, nor any of 
+the other input values. The evaluation of an expression observes the concept of [referential
 transparency][referential-transparency-wiki]. The output of a CESQL expression evaluation is always a boolean _true_ or
-_false_ and it might include an error.
+_false_, and it might include an error.
 
 The CloudEvents Expression Language doesn't support the handling of the data field of the CloudEvent instances, due to
-its polymorphic nature and complexity. We strongly encourage users that needs this functionality to use other more
+its polymorphic nature and complexity. We strongly encourage users that need this functionality use other more
```

Comment:
It is STRONGLY RECOMMENDED that users who need this...

Reply:
just trying to remove the use of "we"

Reply:
As I understand the context, CESQL simply won't filter on the contents of `data`, so any implication that a user could pick CESQL (but shouldn't because we don't "recommend" it) seems misleading. Rewritten.

(Personally I don't get it. What's harder about letting me access fields in data? They could be missing sometimes, but that's already handled for the standard-but-optional fields.)